### PR TITLE
Keyboard nav in edit mode

### PR DESCRIPTION
**Pull request by etokareva**
Also I wanted to change shortcut for Enter? or Tab? to smth like model.navigation().active.cell? model.navigation().active.cell.column.editorSubmitRequest : 'tab', to make it depend on editor-submit-request, but I'm not sure in this. Maybe we should redesign commands...

### CODE REVIEW

File: src/core/column-type/number.column.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ TemplatePath.register('number-cell-edit', (template, column) => {
 class NumberColumnModel extends DataColumnModel{
 	constructor(){
 		super('number');
+		this.editorSubmitShortcut = 'tab';
```
Comment (klumba12) [inline src/core/column-type/number.column.js]: I think we will put to the edit model, and remove from column. It doesn't look natural here. Can you put it to
model.edit().commitShortcuts = {
'$default': 'tab|enter',
'date': 'enter',
...
};

something like that? or maybe in navigation model?

File: src/core/edit/edit.cell.view.js [hunk 2]
```diff
@@ -34,20 +34,19 @@ export default class EditCellView {
 				canExecute: cell => {
 					cell = cell || model.navigation().active.cell;
 					if (cell && this.mode !== 'edit' && model.edit().mode === 'cell' && cell) {
-						return cell.column.canEdit && model.edit().enter.canExecute(this.contextFactory(cell));
+						return cell.column.canEdit && model.edit().enter.canExecute(this.contextFactory(cell)) && model.edit().editMode === 'view';
 					}
 
-					return false;
+					return model.edit().editMode === 'view';
```
Comment (klumba12) [inline src/core/edit/edit.cell.view.js]: why?

Comment (etokareva) [reply src/core/edit/edit.cell.view.js]: You're right. Here we don't need it.

File: src/core/edit/edit.cell.view.js [hunk 3]
```diff
@@ -161,6 +160,21 @@ export default class EditCellView {
 		this._value = value;
 	}
 
+	get commitShortcut() {
+		const commitShortcuts = {
```
Comment (klumba12) [inline src/core/edit/edit.cell.view.js:164]: Kate, can you put to to the edit.model?

File: src/ng/components/column/column.js [hunk 4]
```diff
@@ -31,6 +31,9 @@ class Column extends Component {
 				this.key = '$default';
 			}
 		}
+		if ($attrs.hasOwnProperty('editorSubmitShortcut')) {
```
Comment (klumba12) [inline src/ng/components/column/column.js:34]: lets remove that for now

**Review by klumba12 — APPROVED** [review]

**Pull request merged**